canonical-web-and-design/microk8s.io
# Fix the copyable code patterns

## Done
Fix the copyable code patterns
Remove cssnano
Applied the new docs layout styling
Fixed the footer styling
Update the year in the footer

## QA
- Run `./run clean`
- Run `./run`
- Check it all builds with no errors
- Go to http://0.0.0.0:8027/
- Check the header logo on hover stays white
- Check the code snippets are copyable
- Check the footer looks ok and has the correct year
- Go to docs and check they all look good
- Check the docs footer has the correct year
